packages/ember-testing/lib/support.js

```diff
@@ -19,13 +19,14 @@ const $ = jQuery;
 */
 function testCheckboxClick(handler) {
   let input = document.createElement('input');
-  $(input)
-    .attr('type', 'checkbox')
-    .css({ position: 'absolute', left: '-1000px', top: '-1000px' })
-    .appendTo('body')
-    .on('click', handler)
-    .trigger('click')
-    .remove();
+  input.setAttribute('type', 'checkbox');
```

Comment:
I think this should stay back in jQuery mode since its only ran behind a `!jQueryDisabled` guard